engine/src/flutter/shell/platform/common/geometry.h

```diff
@@ -45,6 +45,7 @@ class Size {
   bool operator==(const Size& other) const {
```

Comment:
This should probably be removed. We're not (or shouldn't) use this in embedders.